actions/admin/access_keys.go

```diff
@@ -16,13 +16,24 @@ import (
 // accessKeysSearch will fetch a list of access keys filtered by metadata
 // Access Keys Search godoc
 // @Summary		Access Keys Search
-// @Description	Access Keys Search
+// @Description	Fetches a list of access keys filtered by metadata, creation range, and other parameters.
 // @Tags		Admin
 // @Produce		json
-// @Param		SearchAccessKeys body filter.AdminSearchAccessKeys false "Supports targeted resource searches with filters and metadata, plus options for pagination and sorting to streamline data exploration and analysis"
-// @Success		200 {object} []response.AccessKey "List of access keys"
-// @Failure		400	"Bad request - Error while parsing SearchAccessKeys from request body"
-// @Failure 	500	"Internal server error - Error while searching for access keys"
+// @Param		xpubId query string false "ID of the xPub associated with the access keys"
+// @Param		includeDeleted query boolean false "Whether to include deleted access keys"
+// @Param		createdRange[from] query string false "Start of creation date range (ISO 8601 format)"
+// @Param		createdRange[to] query string false "End of creation date range (ISO 8601 format)"
+// @Param		updatedRange[from] query string false "Start of last updated date range (ISO 8601 format)"
+// @Param		updatedRange[to] query string false "End of last updated date range (ISO 8601 format)"
+// @Param		revokedRange[from] query string false "Start of revoked date range (ISO 8601 format)"
+// @Param		revokedRange[to] query string false "End of revoked date range (ISO 8601 format)"
+// @Param		page query integer false "Page number for pagination"
+// @Param		pageSize query integer false "Number of results per page"
+// @Param		orderByField query string false "Field to order results by (e.g., 'created_at')"
+// @Param		orderByDirection query string false "Direction of ordering: 'asc' or 'desc'"
```

Comment:
I believe that we already have common way to do this in user API by replacing all those lines with:
```suggestion
// @Param		SwaggerCommonParams query swagger.CommonFilteringQueryParams false "Supports options for pagination and sorting to streamline data exploration and analysis"
```

Please check

Reply:
@dorzepowski sure i can change this but then u still will have same issue 
`// @Param		SwaggerCommonParams query swagger.CommonFilteringQueryParams false "Supports options for pagination and sorting to streamline data exploration and analysis"`

`TypeError: Failed to execute 'fetch' on 'Window': Request with GET/HEAD method cannot have body.`